ext/drreg/drreg.c

```diff
@@ -235,6 +261,9 @@ restore_reg(void *drcontext, per_thread_t *pt, reg_id_t reg, uint slot,
         dr_spill_slot_t DR_slot = (dr_spill_slot_t)(slot - ops.num_spill_slots);
         dr_restore_reg(drcontext, ilist, where, reg, DR_slot);
     }
+    if (release && TEST(DRREG_HANDLE_MULTI_PHASE_SLOT_RESERVATIONS, pt->bb_props)) {
+        instr_set_note(instr_get_prev(where), (void *)(ptr_uint_t)slot);
```

Comment:
Ditto

Reply:
Added the comment, and the proper note usage.